### DISCUSSION TITLE
AI 应用层的壁垒在哪里 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#23] [General]
# AI 应用层的壁垒在哪里 | 高策

江湖小虾米

https://gaocegege.com/Blog/genai/ai-hype

**Comment by Renkai** [comment]
Agent 是啥, 突然在最后一段出现的词🥴

**Reply by hongtaoh** [reply]
我的理解是，Agent 是那种有自主学习能力的模型，它可以自己优化自己。比如，一个可以优化自己的对话模型。但这个需要机器学习的框架进一步升级才能达到，目前来看没有人工干预的话，算法迭代升级比较困难。

**Reply by gaocegege** [reply]
不好意思错过了通知，最终形态的 Agent 我也认同是自主学习，但是目前工业界里常说的 Agent 我认为就是能够接受环境的输入，包括但不限于格式化和非格式化的输入等，然后根据一系列既定的规则和目前模型的能力来给出一个相对智能的输出。学习的过程可能是离线的，收集到足够多的数据后进行 fine tune 等等。
